internal/models/startup.go

```diff
@@ -0,0 +1,9 @@
+package models
+
+type Startup struct {
+	ID            uint   `gorm:"primarykey" json:"id"`
+	Name          string `gorm:"not null" json:"name"`
+	Description   string `gorm:"not null" json:"description"`
+	Image         string `gorm:"not null" json:"image"`
+	GroupedByYear string `gorm:"not null" json:"grouped_by_year"`
```

Comment:
This is quite a strange name for this variable. What does the `GroupedBy` part imply?